src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -248,43 +254,96 @@ class GitOpsConfiguration extends Component<GitOpsProps, GitOpsState> {
         return isInvalid;
     }
 
+    suggestedValidGitOpsUrl() {
+        const gitOpsUrl = this.state.form.host
```

Comment:
Can use this directly everywhere `this.state.form.host` no need to keep it in const unless you are evaluating something